src/cli/cli.cpp

```diff
@@ -1058,6 +1059,107 @@ void Interpreter::ProcessPollPeriod(int argc, char *argv[])
     AppendResult(error);
 }
 
+void Interpreter::ProcessPromiscuous(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+
+    if (argc == 0)
+    {
+        if (otIsLinkPromiscuous() && otPlatRadioGetPromiscuous())
+        {
+            sServer->OutputFormat("true\r\n");
```

Comment:
Would be better to have the status getter/setter match (e.g. `enable/disable` or `true/false`).